ampersand-select-view.js

```diff
@@ -243,6 +257,20 @@ SelectView.prototype.getOptionValue = function (option) {
     return option;
 };
 
+SelectView.prototype.getOptionDisabled = function (option) {
+    if (Array.isArray(option)) return;
```

Comment:
If you'll take a look at `getOptionValue` and `getOptionText`, you'll see that it should return `option[1]`;